plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -130,13 +140,131 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
                 return false;
 
             case SUPPORTS_CANCELLATION:
+            case SUPPORTS_UPDATE:
                 return true;
 
             default:
                 return super.hasBehavior(connectorBehavior);
         }
     }
 
+    @Override
+    public void testUpdate()
+    {
+        // As first step we support only constant updates, remove override after we will be able to support updates with column expressions
```

Comment:
I think we need to have `SUPPORTS_ROW_LEVEL_UPDATE` so we could have dedicated tests for metadata update in `BaseConnectorTest`. Any connector that would implement the new SPI should work the same way.